src/less/core/description-list.less

```diff
@@ -20,6 +20,8 @@
 @description-list-line-border:                  #ddd;
 @description-list-line-color:                   #999;
 
+@description-list-term-weight:                  @base-body-font-weight;
```

Comment:
Meh, I guess this is along the same lines of smarter defaults instead of setting this to `normal` and then have the default, gradient and almost-flat themes do it "the right way" by variabilizing this.